composer.json

```diff
@@ -58,5 +60,21 @@
         "branch-alias": {
             "dev-master": "1.0.x-dev"
         }
+    },
+    "scripts": {
+        "pre-commit": [
+            "vendor/bin/check-syntax-php.sh",
+            "vendor/simplesamlphp/simplesamlphp-test-framework/bin/check-syntax-json.sh",
+            "vendor/simplesamlphp/simplesamlphp-test-framework/bin/check-syntax-xml.sh",
+            "vendor/simplesamlphp/simplesamlphp-test-framework/bin/check-syntax-yaml.sh",
```

Comment:
All syntax checkers are available directly under `vendor/bin`

Reply:
Thanks. I  did c&p from travis.yaml, just to test everything locally.